indico/web/forms/fields.py

```diff
@@ -844,6 +838,8 @@ class IndicoProtectionField(IndicoEnumRadioField):
     radio_widget = JinjaWidget('forms/radio_buttons_widget.html', orientation='horizontal', single_kwargs=True)
 
     def __init__(self, *args, **kwargs):
+        self.protected_object = kwargs.pop('protected_object')(kwargs['_form'])
+        self.acl_message_url = kwargs.pop('acl_message_url')(kwargs['_form'])
```

Comment:
Are we really gonna have a **protection mode** field without an **Access control list
** field?